.github/workflows/ci-checks.yml

```diff
@@ -0,0 +1,29 @@
+name: Flutter Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Setup Flutter Environment
+        uses: subosito/flutter-action@v1
+        with:
+          channel: 'stable'
+          flutter-version: '3.3.10'
+      - name: Get Flutter Dependencies
+        run: flutter pub get
+      - name: Unit Test
+        run: flutter test ${GITHUB_WORKSPACE}/test/chargebee_test.dart
```

Comment:
Can we make this generic to include all files inside the test folder?

Reply:
Yes, Done!